src/applications/ivc-champva/10-10D/tests/e2e/fixtures/data/vet-2a-spouse-child-medabd-ohi.json

```diff
@@ -95,7 +98,7 @@
             "uploaded": false
           }
         ],
-        "sharesAddressWith": "{\"originatorName\":\"Applicant  Jones \",\"originatorAddress\":{\"view:militaryBaseDescription\":{},\"country\":\"USA\",\"street\":\"123 App1 Street\",\"city\":\"App1 Town\",\"state\":\"MD\",\"postalCode\":\"12312\"},\"displayText\":\"123 App1 Street\"}",
+        "sharesAddressWith": "not-shared",
```

Comment:
This data in the JSON was causing the test to be flaky, so replaced with an (equally valid) simpler value `not-shared`.

Reply:
Is the applicant not sharing addresses with the spouse or child?